src/display/text_layer.js

```diff
@@ -469,6 +462,15 @@ class TextLayer {
     return canvasContext;
   }
 
+  static #ensureCtxFont(ctx, size, family) {
+    const cached = this.#canvasCtxFonts.get(ctx);
+    if (cached && size === cached.size && family === cached.family) {
+      return; // The font is already set.
+    }
+    ctx.font = `${size}px ${family}`;
+    this.#canvasCtxFonts.set(ctx, { size, family });
```

Comment:
nit: with the above change, just set the new size and family in `cached` to avoid to have to recreate a new structure.

Reply:
Done; since I agree that avoiding extra Object creation is generally a good thing.